### PR TITLE
AE-2838 QNH Difference During Approach restricted to low altitudes

### PR DESCRIPTION
**Pull request by gille02d**
The KPV will only trigger where Altitude AAL reached 100 ft. This ensures
we have an Altitude AAL which is quite representative of the height above
the runway.

### CODE REVIEW

File: analysis_engine/key_point_values.py [hunk 1]
```diff
@@ -6609,14 +6609,11 @@ def derive(self, alt_mcp=P('Altitude Selected (MCP)'),
 class QNHDifferenceDuringApproach(KeyPointValueNode):
     '''
     This KPV calculates the QNH difference between actual QNH and expected
-    QNH during approach when crossing 100 ft AAL in descent or at the minimum
-    altitude reached (due to go around) whichever is greater. 100 ft altitude
+    QNH during approach when crossing 100 ft AAL in descent. 100 ft altitude
```
Comment (kirku1g) [inline analysis_engine/key_point_values.py]: While we're changing the docstring, I don't think "100 ft altitude" following the sentence helps.

Comment (gille02d) [reply analysis_engine/key_point_values.py]: Absolutely! Thanks for re-reading the docstring.

I've reworded and I hope it will make more sense.

**Review by kirku1g — DISMISSED** [review]
LGTM.

**Review by kirku1g — APPROVED** [review]
LGTM.

**Pull request merged**